lang/cs/Org.Apache.REEF.Client.Tests/WindowsYarnJobCommandProviderTests.cs

```diff
@@ -50,9 +50,7 @@ public class WindowsYarnJobCommandProviderTests
             "%HADOOP_HOME%/share/hadoop/hdfs/*",
             "%HADOOP_HOME%/share/hadoop/hdfs/lib/*",
             "%HADOOP_HOME%/share/hadoop/mapreduce/*",
-            "%HADOOP_HOME%/share/hadoop/mapreduce/lib/*",
-            "reef/local/*",
```

Comment:
They are not gone. They are still there in the `expectedCommand`. They are just explicitly added as classpath. Everything except the REEF folders come from `IYarnCommandLineEnvironment`. This is just setting up that portion.